### PR TITLE
Formatting fixes

### PR DESCRIPTION
**Pull request by ghost**
@ahardin-rh Can you peer review this PR? This is the issue we discussed on IRC. Thanks!

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@vikram-redhat Fixing a formatting issue for 3.6. This corresponds with PRs 6493, 6511, 6512, and 6513.

File: scaling_performance/optimizing_storage.adoc [hunk 1]
```diff
@@ -30,22 +30,21 @@ a|* Presented to the operating system (OS) as a block device
...
 * Concurrency, latency, file locking mechanisms, and other capabilities vary widely between protocols, implementations, vendors, and scales.
-| CNS/CRS GlusterFS <1>, RHEL NFS, NetApp NFS <2>, Azure File, Vendor NFS, Vendor GlusterFS <3>, Azure File, AWS EFS
+| CNS/CRS GlusterFS (1), RHEL NFS, NetApp NFS (2), Azure File, Vendor NFS, Vendor GlusterFS (3), Azure File, AWS EFS
 
 | Object
 a| * Accessible through a REST API endpoint
 * Configurable for use in the {product-title} Registry
 * Applications must build their drivers into the application and/or container.
-| CNS/CRS GlusterFS <1>, Ceph Object Storage (RADOS Gateway), OpenStack Swift, Aliyun OSS, AWS S3, Google Cloud Storage, Azure Blob Storage, Vendor S3 <3>, Vendor Swift <3>
+| CNS/CRS GlusterFS (1), Ceph Object Storage (RADOS Gateway), OpenStack Swift, Aliyun OSS, AWS S3, Google Cloud Storage, Azure Blob Storage, Vendor S3 (3), Vendor Swift (3)
```
Comment (ahardin-rh) [inline scaling_performance/optimizing_storage.adoc]: The formatting of <1>, <2>, <3> is actually correct. I think we just need to add an "a" before the cell to get this to render properly. For example:

    a|CNS/CRS GlusterFS <1>, RHEL NFS, NetApp NFS <2>, Azure File, Vendor NFS, Vendor GlusterFS <3>, Azure File, AWS EFS

Give that a try and see if it builds better.

File: scaling_performance/optimizing_storage.adoc [hunk 2]
```diff
@@ -30,46 +30,39 @@ a|* Presented to the operating system (OS) as a block device
 bypassing the file system
 * Also referred to as a Storage Area Network (SAN)
 * Non-shareable, which means that only one client at a time can mount an endpoint of this type
-| CNS/CRS GlusterFS <1> iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, AWS EBS <1>, Dell/EMC Scale.IO,
-VMware vSphere Volume, GCE Persistent Disk <1>, Azure Disk
+| CNS/CRS GlusterFS footnoteref:[dynamic PV,CNS/CRS GlusterFS, Ceph RBD, OpenStack Cinder, AWS EBS, Azure Disk, GCE persistent disk, and VMware vSphere support dynamic persistent volume (PV) provisioning natively in {product-title}.]  iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, AWS EBS footnoteref:[dynamic PV], Dell/EMC Scale.IO, VMware vSphere Volume, GCE Persistent Disk footnoteref:[dynamic PV], Azure Disk
```
Comment (vikram-redhat) [inline scaling_performance/optimizing_storage.adoc]: Hey @tmorriso-rh and @openshift/team-documentation - it seems that the portal doesn't support space in the footnoteref. So this should be dynamicPV and not dynamic PV. I have made the fix for this and other footnotes via https://github.com/openshift/openshift-docs/pull/6597.

@tmorriso-rh - can you please add this in our docs guidelines?

Comment (tmorriso-rh) [reply scaling_performance/optimizing_storage.adoc]: @vikram-redhat Hi Vikram, I added this information to the quick markup reference table. Please revise as necessary. Thanks.

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

**Pull request merged**